### PR TITLE
added TS130F Dual curtain/blind module definition

### PR DESCRIPTION
**Pull request by dominicusmento**
added definition for dual curtain blind module.. after checking it in this issue: 
https://github.com/Koenkk/zigbee2mqtt/issues/8705

### CODE REVIEW

**Comment by dominicusmento** [comment]
it's a duplicate model but very different device, with dual curtain functionality and different manufacturer: _TZ3000_j1xl73iw

File: devices/lonsonho.js [hunk 1]
```diff
@@ -32,6 +32,26 @@ module.exports = [
             exposes.enum('backlight_mode', ea.ALL, ['LOW', 'MEDIUM', 'HIGH']),
             exposes.numeric('calibration_time', ea.STATE).withUnit('S').withDescription('Calibration time')],
     },
+    {
+        fingerprint: [{modelID: 'TS130F', manufacturerName: '_TZ3000_j1xl73iw'}],
+        model: 'TS130F_dual',
```
Comment (Koenkk) [inline devices/lonsonho.js:37]: Is there any model number written on the device? If yes please use that, and rename the picture/md for the docs to this.

Comment (dominicusmento) [reply devices/lonsonho.js:37]: Unfortunately no, I've checked it now.. all the other sides of the device are empty.. manual also doesn't specify any model number.. It's called '2CH Zigbee curtain module' even in manual.. App in manual is Tuya, I'm guessing it's Lonsonho only by seller mentioning that info on [aliexpress](https://www.aliexpress.com/item/1005003044227983.html)..
image
manual:
image

Comment (Koenkk) [reply devices/lonsonho.js:37]: ok thanks

**Comment by Koenkk** [comment]
thanks!

**Pull request merged**